sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -146,12 +150,12 @@ const MarketDetails: React.FC<MarketDetailsProps> = ({ baseCurrencyKey }) => {
 				),
 			},
 			'Funding Rate': {
-				value: marketSummary?.currentFundingRate
-					? formatPercent(marketSummary?.currentFundingRate ?? zeroBN, { minDecimals: 6 })
+				value: avgFundingRate
+					? formatPercent(avgFundingRate ?? zeroBN, { minDecimals: 6 })
 					: NO_VALUE,
```

Comment:
This should still fallback to `marketSummary?.currentFundingRate` IMO in case two events in the period aren't available. In which case it should also add some indication that a different value is being used (e.g. `(inst)` suffix) or something.

Reply:
These issues will be addressed in Issue #643